### ISSUE TITLE
[Supporting the platform - Logic Apps - Overview]: Horizontal scroll bar appears on 'Overview' screen after setting viewport to 320*256 pixels.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6104]
### Describe the issue 

User Experience:

 Low-vision and cognitive users will not be able to navigate through all the controls properly and will get challenges while navigating on the controls present on the page as the user would need to scroll a lot due to two-dimensional scrolling.

 Note: User credentials should NOT be included in the bug.

Pre- requisite

 Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

Repro Steps:

- Open [https://ms.portal.azure.com/](<https://ms.portal.azure.com/>) and login via valid credentials. 
- Navigate to the Left Nav bar and select 'More Services', search 'Logic apps'. 
- Select any created resource. 
- Select Overview in left navigation pane. 
- Navigate all the controls on the blade like Top menu bar (Run Trigger, Refresh, Edit , Delete) , blade content , Zoom- in\out.. etc 
- Verify whether the Horizontal scroll bar appears on 'Overview' screen after setting viewport to 320\*256 pixels or not. 

Actual Result:

 Horizontal scroll bar appears on 'Overview' screen after setting viewport to 320\*256 pixels. All controls are not visible without scrolling the page horizontally.

Expected Result:

 Horizontal scroll bar should not appear on 'Overview' screen after setting viewport to 320\*256 pixels.

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by ccastrotrejo** [comment]
PR in portal repo to avoid horizontal scroll https://msazure.visualstudio.com/One/_git/AzureUX-LogicAppsPortal/pullrequest/11482873

**Comment by ccastrotrejo** [comment]
Closing this one as PR has been merged into master
